libraries/provision/ansible/playbooks/sgcollect-info.yml

```diff
@@ -33,6 +30,3 @@
 
   - include: tasks/sa-sgcollect-info-windows.yml
     when: ansible_os_family == "Windows"
-
-  - include: tasks/sa-pullRedacted-zip-windows.yml
```

Comment:
same as above

Reply:
pullling redacted zip file is part of  sg-sgcollect-info-windows.yml, so it is removed.